RELEASING.md

```diff
@@ -0,0 +1,232 @@
+## Releasing
```

Comment:
This and all subsequent headers can be moved up a level (i.e., drop a `#` in each).